packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -200,7 +201,10 @@ export const RestrictedApplication = props => (
                       </ToggleFeature>
 
                       <header>
-                        <AppBar user={user} />
+                        <AppBar
+                          user={user}
+                          projectKeyFromUrl={projectKeyFromUrl}
```

Comment:
Passing it in feels easier than reading there again using the utils especially for testing.